src/recipe/Baked.Recipe.Service.Application/Runtime/RuntimeLayer.cs

```diff
@@ -11,6 +12,8 @@ namespace Baked.Runtime;
 
 public class RuntimeLayer : LayerBase<BuildConfiguration, AddServices, PostBuild>
 {
+    public const string FILE_PROVIDERS_KEY = "FILE_PROVIDERS_KEY";
```

Comment:
this is public, but no other class uses it, inline this const

Reply:
we used this const key in `AddFileProvider` extensions